base/intrinsics/intrinsics.odin

```diff
@@ -71,8 +71,12 @@ prefetch_write_data        :: proc(address: rawptr, #const locality: i32 /* 0..=
 // Compiler Hints
 expect :: proc(val, expected_val: T) -> T ---
 
-// Linux and Darwin Only
-syscall :: proc(id: uintptr, args: ..uintptr) -> uintptr ---
+// Linux, Darwin, and FreeBSD Only
+when ODIN_OS == .FreeBSD {
+	syscall :: proc(id: uintptr, args: ..uintptr) -> (uintptr, bool) ---
+} else {
+	syscall :: proc(id: uintptr, args: ..uintptr) -> uintptr ---
+}
```

Comment:
I don't like that they have different return values.

Why not have a separate intrinsic that is freebsd specific? e.g. `intrinsics.freebsd_syscall`

Reply:
I was uncertain about this myself back when I wrote this, but I can do that instead. Separate intrinsics is probably for the best.